rio_cogeo/cogeo.py

```diff
@@ -56,9 +56,10 @@ def cog_translate(
             meta["count"] = len(indexes)
             meta.pop("nodata", None)
             meta.pop("alpha", None)
+
+            meta.update(**dst_kwargs)
             meta.pop("compress", None)
             meta.pop("photometric", None)
-            meta.update(**dst_kwargs)
```

Comment:
we still need the `memory` file to be `tiled` to do window read/write and `dst_kwargs` should alway be at least: 
```
    defaults = {
        "driver": "GTiff",
        "interleave": "pixel",
        "tiled": True,
        "blockxsize": 512,
        "blockysize": 512,
    }
```